__tests__/lint/operation-security-scheme-oas3-error/snapshot.js

```diff
@@ -100,25 +100,35 @@ Property \`authorizationUrl\` is not expected here.
 Error was generated by the spec rule.
 
 
-[8] openapi.yaml:1:1 at #/
+[8] openapi.yaml:1:1 at #/openapi
 
 Every API should have security defined on the root level or for each operation.
 
- 1 | openapi: 3.0.0
-   | ^^^^^^^^^^^^^^
- 2 | servers:
-   | ^^^^^^^^
- … | < 55 more lines >
-58 |       authorizationUrl: 'http://petstore.swagger.io/api/oauth/dialog'
-   |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-59 |
+1 | openapi: 3.0.0
+  | ^^^^^^^
```

Comment:
Highlighting `openapi` sounds misleading to me. Wouldn't it be enough to highlight only operations that miss the security field and put there a message that it could be set on the definition's root as well?
@RomanHotsiy  it's most likely a question to you 🙂

Reply:
yes, let's highlight operations only.

Reply:
It will also allow someone to know how many operations are unsecured more clearly in combination with the summary format.